pyproject.toml

```diff
@@ -25,7 +25,7 @@ keywords = ["ISPyB", "database"]
 requires-python = ">=3.7"
 readme = "README.md"
 license = { text = "Apache License 2.0" }
-dependencies = ["mysql-connector-python>=8.0.32", "sqlalchemy<2", "tabulate"]
+dependencies = ["mysql-connector-python>=8.0.32", "sqlalchemy>=2", "tabulate"]
```

Comment:
What happens if there is an sqlalchemy version 3? Should we specify <3 as well?

Reply:
Yes, you're right we should. I've done that now